app/models/zillow.rb

```diff
@@ -19,22 +19,29 @@ def initialize(attrs)
     @type         = attrs["searchresults"]["response"]["results"]["result"]["useCode"]
     @neighborhood = attrs["searchresults"]["response"]["results"]["result"]["localRealEstate"]["region"]["name"]
     @sqft         = attrs["searchresults"]["response"]["results"]["result"]["finishedSqFt"]
-  end
+    @rent         = attrs["searchresults"]["response"]["results"]["result"]["rentZestimate"]
+    @sold_price   = attrs["searchresults"]["response"]["results"]["result"]["lastSoldPrice"]["__content__"]
+    @sold_date    = attrs["searchresults"]["response"]["results"]["result"]["lastSoldDate"]
```

Comment:
Dealt with.

2016-02-02 14:32 GMT-08:00 Bookis Worthy <notifications@github.com>:

> In app/models/zillow.rb
> <https://github.com/smurfik/auction_house/pull/23#discussion_r51647398>:
>
> > @@ -19,22 +19,29 @@ def initialize(attrs)
> >      @type         = attrs["searchresults"]["response"]["results"]["result"]["useCode"]
> >      @neighborhood = attrs["searchresults"]["response"]["results"]["result"]["localRealEstate"]["region"]["name"]
> >      @sqft         = attrs["searchresults"]["response"]["results"]["result"]["finishedSqFt"]
> > -  end
> > +    @rent         = attrs["searchresults"]["response"]["results"]["result"]["rentZestimate"]
> > +    @sold_price   = attrs["searchresults"]["response"]["results"]["result"]["lastSoldPrice"]["__content__"]
> > +    @sold_date    = attrs["searchresults"]["response"]["results"]["result"]["lastSoldDate"]
>
> When I do a search for my house (4318 4th Ave NE) I get an error on line
> 23 because it doesn't have the lastSoldPrice, could you wrap that in a
> conditional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/smurfik/auction_house/pull/23/files#r51647398>.
>



-- 
Karen Hambro